### PR TITLE
Update AWS_REGION if set in .aws/config

### LINKED ISSUE #1: Also change `AWS_REGION` region based on aws config profile
Thanks for building this great tool.

I would like to extend it to also change `AWS_REGION` environment variable based on the profile selected. 

e.g. in `.aws/config`

```
[profile region1]
region=ap-southeast-2
```

```
[profile region2]
region=us-east-1
```

Thoughts?

### PR DESCRIPTION
**Pull request by dennisroche**
If `region` is configured in an AWS profile, then update `AWS_REGION` environment variable

Closes #1

### CODE REVIEW

**Comment by dennisroche** [comment]
This is the minimal change, however refactored further in https://github.com/dennisroche/posh-awsp/pull/1/files

All test pass.

File: posh-awsp.psm1 [hunk 1]
```diff
@@ -287,8 +392,9 @@ function Read-MenuSelection {
   #
   for($i = 0; $i -lt $Items.Length; $i++) {
     $Indicator = if ($CurrentIndex -eq $i) { "*" } else { " " }
+    $Name = if ($CurrentIndex -eq $i) { "`e[36m$($Items[$i])`e[0m" } else { "$($Items[$i])" }
```
Comment (dennisroche) [inline posh-awsp.psm1]: Using ANSI colors to highlight the selected profile. https://www.reddit.com/r/PowerShell/comments/b06gtw/til_that_powershell_can_do_colors/

Not required change and can revert. Looks cool tho.

File: posh-awsp.psm1 [hunk 2]
```diff
@@ -21,13 +21,13 @@ function Get-AWSConfigFile {
     # If the environment variable is not configured, calculate
     # the default location as documented in the AWS CLI docs.
     #
-    $Home = $Env:HOME
-    
+    $AwsHome = $Env:HOME
```
Comment (dennisroche) [inline posh-awsp.psm1:24]: `Home` conflicted with a default environment variable. Renamed.

**Comment by dennisroche** [comment]
Removed setting of env variable as mentioned in https://github.com/jonscheiding/posh-awsp/issues/1. I think keeping the region name in the profile listing is useful - up to you.

**Comment by jonscheiding** [comment]
@dennisroche This is now pushed to the Gallery as version 0.8.0. Thanks!

**Pull request merged**